packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -147,28 +162,42 @@ class DebuggerScreenBodyState extends State<DebuggerScreenBody>
       },
     );
 
-    return Split(
-      axis: Axis.horizontal,
-      initialFractions: const [0.25, 0.75],
-      children: [
-        OutlinedBorder(child: debuggerPanes()),
-        Column(
-          children: [
-            DebuggingControls(controller: controller),
-            const SizedBox(height: denseRowSpacing),
-            Expanded(
-              child: Split(
-                axis: Axis.vertical,
-                initialFractions: const [0.74, 0.26],
+    return Shortcuts(
+      shortcuts: <LogicalKeySet, Intent>{
```

Comment:
These shortcuts seem to be ignored unless I manually focus something in the debugger pane; the only candidate I've found that works is the filter textbox itself which kind of defeats the purpose.

Reply:
Odd. If as a hack you place the shortcuts widget at the very root of the application do you get the expected behavior? The problem might be that the default focus is on the tab bar or some other portion of the UI outside the debugger screen.

Reply:
Yes, it does work if I promote the shortcuts widget to the root of the application. I think that you are probably right about the focus being outside the debugger screen

Reply:
lets enable devtools screens to attach keyboard handlers placed at the root of the widget tree. seems reasonable for screens to always receive keyboard events when their screen is open.

Reply:
I'll play around with that.

I currently have a theory that the way tab bodies are rendered by the devtools doesn't play nice with autofocus. It would explain a few things. I did verify that the primary focus continues to be the InspectorTree from the last tab when I want it to be in the debugger, and autofocus properties don't change that.

Reply:
Got it! The secret is that every screen needs its own FocusScope, then things like autofocus work correctly. Thanks to @gspencergoog for helping me solve it.